front-end/src/components/GovernanceCard.tsx

```diff
@@ -35,6 +38,10 @@ const GovernanceAction = function ({
 				</Segment>
 				<Segment>
 					<h4 className={'proposalTitle'}>{title}</h4>
+					<CreationLabel
+						address={address}
+						text={'proposed'}
```

Comment:
This proposed is weird IMO.
Unless you see an easy way to just add a "by" in front of the inline user. I'd rather have nothing than a "proposed" alone.

This also asks the question if all of this should be done in the same component. It ends up being an "AddressInline" rather than a `CreationLabel`. What do you think?

Reply:
Hmm, yeah, `CreationLabel` might get a bit bloated. Moving Address component directly in the GovernanceCard without text for now (as we're redoing the GovCard anyway).

Reply:
Actually, Typescript yells as the address prop in the` Address` component isn't optional, and it probably should stay that way.